app/controllers/concerns/patient_referral.rb

```diff
@@ -27,8 +27,7 @@ def load_index_vars
     else
       @patient_referrals = Health::PatientReferral.none
     end
-    @patient_referrals = @patient_referrals.
-      page(params[:page].to_i).per(20)
+    @pagy, @patient_referrals = pagy(@patient_referrals, items: 3)
```

Comment:
@gigxz was this an intentional change to 3 per page or just for testing?

Reply:
oops didnt mean to leave that - good catch, I'll fix and search for others